pihole

```diff
@@ -425,7 +425,7 @@ tricorderFunc() {
 }
 
 updateCheckFunc() {
-  "${PI_HOLE_SCRIPT_DIR}"/updatecheck.sh "$@"
+  "${PI_HOLE_SCRIPT_DIR}"/updatecheck.sh "$1"
```

Comment:
If we accept my suggestion for `shift`ing then this can remain `@` to allow `updatecheck.sh` to have multiple arguments passed to it in the future.

Reply:
OK.

Reply:
I decided to use `shift`.
I kept the `$@` on both cases (Lines 248 and 556) to allow multiple arguments.